kubernetes/scenescape-chart/templates/pgserver-dep.yaml

```diff
@@ -21,145 +21,81 @@ spec:
+                  key: DATABASE_PASSWORD
+            - name: POSTGRES_DB
+              value: "scenescape"
 {{- if .Values.proxy.enabled }}
-          - name: HTTP_PROXY
-            value: {{ .Values.proxy.httpProxy }}
-          - name: HTTPS_PROXY
-            value: {{ .Values.proxy.httpsProxy }}
-          - name: NO_PROXY
-            value: {{ .Values.proxy.noProxy }}
-          - name: http_proxy
-            value: {{ .Values.proxy.httpProxy }}
-          - name: https_proxy
-            value: {{ .Values.proxy.httpsProxy }}
-          - name: no_proxy
-            value: {{ .Values.proxy.noProxy }}
+{{- with .Values.proxy }}
+            - name: HTTP_PROXY
+              value: {{ .httpProxy }}
+            - name: HTTPS_PROXY
+              value: {{ .httpsProxy }}
+            - name: NO_PROXY
+              value: {{ .noProxy }}
+            - name: http_proxy
+              value: {{ .httpProxy }}
+            - name: https_proxy
+              value: {{ .httpsProxy }}
+            - name: no_proxy
+              value: {{ .noProxy }}
+{{- end }}
 {{- end }}
           imagePullPolicy: Always
           securityContext:
             privileged: true
             capabilities:
               add: ["SYS_ADMIN"]
           ports:
-          - containerPort: 5432
-          readinessProbe:
-            exec:
-              command:
-              - cat
-              - /tmp/healthy
-            periodSeconds: 1
-          resources: {}
+            - containerPort: 5432
           volumeMounts:
-          - mountPath: /dev/fuse
-            name: dev-fuse
-          - mountPath: /run/secrets/django
-            name: django
-            readOnly: true
-          - mountPath: /run/secrets/controller.auth
-            name: controller-auth
-            readOnly: true
-            subPath: controller.auth
-          - mountPath: /run/secrets/browser.auth
-            name: browser-auth
-            readOnly: true
-            subPath: browser.auth
-          - mountPath: /run/secrets/calibration.auth
-            name: calibration-auth
-            readOnly: true
-            subPath: calibration.auth
-          - mountPath: /run/secrets/scenescape-ca-pem
-            name: scenescape-ca-pem
-            readOnly: true
-            subPath: scenescape-ca.pem
-          - mountPath: /workspace
-            name: pgserver-storage
-            subPath: workspace
-          - mountPath: /workspace/media
-            name: media-storage
-          - mountPath: /home/scenescape/SceneScape/sample_data
-            name: sample-data-storage
-            subPath: sample_data
-          - mountPath: /home/scenescape/SceneScape/user_access_config.json
-            name: user-access-config-storage
-            subPath: user_access_config.json
-          {{- if .Values.tests.enabled }}
-          - mountPath: /workspace/tests
-            name: tests-storage
-            subPath: tests
-          {{- end }}
+            - mountPath: /dev/fuse
```

Comment:
The addition of /dev/fuse as a volume mount is not required for standard PostgreSQL operation. Unless the database needs to interact with a FUSE-based filesystem, this mount can be safely removed.